components/dashboard/src/settings/selectClass.tsx

```diff
@@ -65,9 +65,9 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
                     </SelectableCardSolid>
                     <SelectableCardSolid
                         className="w-36 h-32"
-                        title="XL"
-                        selected={workspaceClass === "XL"}
-                        onClick={() => actuallySetWorkspaceClass("XL")}
+                        title="Large"
+                        selected={workspaceClass === "g1-large"}
```

Comment:
We really need to source these values from config :confused:

Reply:
Yes! All of this should come from config, but for the skateboard I left it at hard coded.

Reply:
> but for the skateboard I left it at hard coded.

Yes, fine in this context. We can have a follow up PR for API + dynamic rendering.